include/deal.II/lac/utilities.h

```diff
@@ -33,6 +36,67 @@ namespace Utilities
   namespace LinearAlgebra
   {
 
+    /**
+     * Return elements of continuous Givens rotation matrix and norm of the input vector.
+     *
+     * That is for a given
+     * pair @p x and @p y, return $c$ , $s$ and $\sqrt{x^2+y^2}$ such that
+     * \f[
+     * \begin{bmatrix}
+     * c  & s \\
+     * -s & c
+     * \end{bmatrix}
+     * \begin{bmatrix}
+     * x \\
+     * y
+     * \end{bmatrix}
+     * =
+     * \begin{bmatrix}
+     * \sqrt{x^2+y^2} \\
+     * 0
+     * \end{bmatrix}
+     * \f]
+     *
+     * @note The function is implemented for real valued numbers only.
+     *
+     * @author Denis Davydov, 2017
+     */
+    template<typename NumberType>
+    std::array<NumberType,3> Givens_rotation(const NumberType &x,
```

Comment:
Do we capitalize proper names in function interfaces elsewhere?

Reply:
in this file  we already have `Lanczos_largest_eigenvalue` and `Chebyshev_filter`, so I would say yes.

Reply:
Alright, I didn't check those, then I guess we keep those names.

Reply:
We're not consistent. For example, we also use `householder` and `gauss_jordan` as lower-case words. I think we used to always use lower-case, but that train has left the station.

Reply:
@kronbichler @bangerth actually, I added `Lanczos_largest_eigenvalue` and `Chebyshev_filter` after the latest release (8.5), sometime in August. So I can easily rename all those to be lower-case.

Do we capitalize elsewhere?

Reply:
I think we've always used lower-case for function names, even for proper names.

Reply:
last commit fixes this.